src/main/resources/jenkins/branch/MultiBranchProject/configure-common.jelly

```diff
@@ -23,17 +23,11 @@
  -->
 <?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form" xmlns:p="/lib/hudson/project" xmlns:this="this">
-  <p:config-disableBuild/>
```

Comment:
> Is there matching pull requests to add these back in?

No, these were all removed intentionally—they should never have been there to begin with. The `disabled` field, for example, is defined in `AbstractProject`, and is not available in `MultiBranchProject`.